client/src/components/Login.js

```diff
@@ -57,18 +57,57 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('');
+    const [password, setPassword]=useState('');
+    const [errors, setErrors]=useState('');
+
+    const validations = () => {
+        let temp = {};
```

Comment:
how about `const errorsCopy = {...errors}` ? We generally want to avoid vague variable names. Then below, `return Object.values(errorsCopy).every(field => field === "")`